ceph-api-nightly/config/definitions/ceph-api-nightly.yml

```
@@ -89,3 +89,8 @@
     publishers:
       - email:
           recipients: ceph-qa@ceph.io
+
+      - archive:
+          artifacts: 'build/out/mgr.x.log'
```

What about this?
```suggestion
          artifacts: 'build/out/mgr.*.log'
```
And perhaps in the future we'd need to add every log... But let's wait till then.